content/docs/setup/kubernetes/platform-setup/minikube/index.md

```diff
@@ -52,14 +53,14 @@ Administrative privileges are required to run minikube.
 
     One effective way to monitor memory usage in minikube:
 
-    {{< text_hack bash >}}
+    {{< text bash >}}
     $ minikube ssh
     $ top
     GiB Mem : 12.4/15.7
-    {{< /text_hack >}}
+    {{< /text >}}
 
-    This shows 12.4GiB used of 15.7 GiB in the virtual machine.  This data was generated with
-    VMWare Fusion hypervisor on a Macbook Pro 13" running Istio 1.2 with bookinfo installed.
+    This shows 12.4GiB used of 15.7 GiB RAMin the virtual machine.  This data was generated with
```

Comment:
Spelling will probably get this, RAMin => RAM in.

Reply:
thanks @ericvn .